src/codemodder/codemods/base_visitor.py

```diff
@@ -66,6 +66,15 @@ def node_position(self, node):
     def lineno_for_node(self, node):
         return self.node_position(node).start.line
 
+    def code(self, node: cst.CSTNode) -> str:
```

Comment:
Just curious but I don't see where this is used in the changes below. Is this just to enable testing?

Reply:
it is used :) it's kinda buried in there isn't it? https://github.com/pixee/codemodder-python/pull/479/files#diff-f5b72205f33eb7f46fdbe920b68b37edc816051d59deec75147453b533e567e8R74